utils/servicioSisa.ts

```diff
@@ -16,7 +17,7 @@ export function getSisaCiudadano(nroDocumento, usuario, clave, sexo) {
         pathSisa += `&sexo=${sexo}`;
     }
     return new Promise((resolve, reject) => {
-        request(pathSisa, (err, response, body) => {
+        request({ uri: pathSisa, rejectUnauthorized: false }, (err, response, body) => {
```

Comment:
user `handleHttpRequest`
```suggestion
        await handleHttpRequest({ uri: pathSisa, rejectUnauthorized: false });
```

Reply:
:+1:

Reply:
Listo!